flutter/lib/models/party.dart

```diff
@@ -22,4 +21,6 @@ class Party {
 
   Member? getMemberAt(Cell cell) =>
       aliveMembers.firstWhereOrNull((m) => m.location == cell);
+
+  bool get isActive => chief.isAlive && !chief.isSurrounded() && movableMembers.isNotEmpty;
```

Comment:
I prefer keeping `isActive` property close to `isLost`, as the logic is very related.